Dockerfile

```diff
@@ -13,6 +13,9 @@ RUN apt-get update && apt-get upgrade && apt-get install -y ${PACKAGES}
 RUN adduser --uid ${UID} ${USERNAME}
 
 FROM scratch AS runner
+LABEL description="This Docker image runs the MultiversX monitoring binary."
+LABEL website="https://multiversx.com/"
+LABEL maintainer="Virgil GAGNIEUX <gagnieux.virgil@proton.me>"
```

Comment:
Since you are the third that changes the code or installation support files, I think this will be better changed to:
```
LABEL maintainer="MultiversX Validators Community <https://t.me/MultiversXValidators>"
```

Reply:
Done 🙂